### DISCUSSION TITLE
Enabled GitHub Copilot Chat for my organisation, but can’t access chat in IntelliJ or even join the waitlist

### DISCUSSION THREAD

**Discussion by Qhaios** [community/community#74941] [Copilot Conversations]
### Select Topic Area

Question

### Body

I just enabled GitHub Copilot chat for my organisation (following the instructions), but in IntelliJ it still says ‘Join Chat in JetBrains Waitlist’. When I click the button, I get

> Oops!
> Admission to the private beta for GitHub Copilot Chat in JetBrains IDEs is limited to users with an active GitHub Copilot Subscription.

yet I’ve been using the standard GitHub Copilot business subscription for months, and it’s still active in my IDE currently

**Comment by imwule** [comment]
Admission to the private beta for [GitHub Copilot Chat](https://github.co/copilot-chat-jetbrains) in JetBrains IDEs is limited to users with an active GitHub Copilot subscription. **Joining the waitlist does not guarantee access.**
Admission to the private beta for GitHub Copilot Chat in JetBrains IDEs will be limited.**You will receive an email if you are granted access.**
so please wait.

**Reply by Qhaios** [reply]
If you look at my post again, you’ll see I said **I can’t even join the waitlist**

**Reply by imwule** [reply]
i am sorry.

**Reply by voorjordanp** [reply]
this private beta is just plain stupid. vs code has this copilot chat already implemented for monthts.
this private beta is just plain stupidity of jetbrains and there whole staff! blatently said but just totally facts..

**Reply by bera-bin-le** [reply]
totally agree, this is ridiculous, to have a waiting list for paying customers to use a paid service plugin.

**Reply by zozoheir** [reply]
After 5 years of Pycharm pro, I'm switching to VSCode due to this. Get your job done or lose customers.

**Comment by techdragon** [comment]
I'm an organisation admin and I can't seem to make it work either. Copilot is enabled, I've gone in and make sure that "GitHub Copilot Chat for IDEs" is turned on as well, in case that was related... nothing. I don't have an "account manager" because I'm not an enterprise customer, so what am I supposed to do here? 

How do I as an organisation admin, join the waitlist either for myself, everyone in the org, or even just a specific person in the org? ... Theres no information besides the stupid landing page.

**Comment by DaltonWorsnup** [comment]


**Reply by leereichardt** [reply]
This comment does not describe the issue at all.

**Comment by twigs67** [comment]
Same issue here. I imagine they're trying to push their Ai assistant. Idk, but it's annoying because it works just fine on Vscode.

**Reply by oerlikonone** [reply]
I have the same issue here with my team. In VSCode it is working out of the box. In IntelliJ it is not working. Very annoying. And I also cannot join the beta version.

**Comment by dio5** [comment]
Subscribe. Would rather not have to switch to vscode so would love to hear the status of this.

**Comment by egor-n** [comment]
I also faced the issue where the Copilot Chat welcome screen was always being displayed. What helped is selecting the `New Chat Session` action directly: `Cmd + Shift + A` -> `New Chat Session`.

**Reply by dio5** [reply]
Thanks for this, this helped!

Though the generate docs/explain this/... from the copilot dropdown still doesn't

**Reply by loadinglucian** [reply]
This fixed it for me too (had to go into Settings > Keymap to track down the New Chat Session command (it didn't have a default shortcut for me) but once I set up a shortcut and fired it off, the "Join the Waitlist" screen went away for good. Ty so much @egor-n

**Reply by RagingLeviathan** [reply]
@egor-n @lucianvacaroiu thank you both so much for these pieces of information! Seems like only the chat itself is working, and at that I've needed to hit new chat command a few times, since the waitlist screen crops up again. Has anyone got any ideas why the dropdown commands from rightclicking on Copilot in a file itself don't seem to work?

**Reply by jorbascrumps** [reply]
It seems the March 4 update (1.4.19.4986) renamed this action to `New Conversation`.

**Reply by dio5** [reply]
@RagingLeviathan since the PhpStorm 2023.3.5 Build #PS-233.14808.18, built on March 11, 2024 upgrade the dropdown commands work for me (I was on a few months older version before).

**Reply by RagingLeviathan** [reply]
@dio5 cheers! I found that they were working for me as well, though note of caution for anyone who may be in a similar situation; I've had to remove Copilot from PhpStorm entirely now, as, for some strange reason, it was somehow responsible for locking my Windows AD account on my work laptop! I've got a manual proxy set up in PhpStorm, to help with connecting to the corporate network, so my current hunch is either PhpStorm or Copilot was doing something weird, as the kinds of errors I was getting from Windows was 'The referenced account is currently locked out and may not be logged on to' when trying to unlock the laptop and ''Your account is temporarily locked to prevent unauthorized use." when attempting to sign in to 365 office via the web!

**Reply by hishigu** [reply]
You saved my life

**Comment by Avec112** [comment]
This is bonkers. _If you have a Copilot Individual subscription, and you would like to use Copilot Chat in a JetBrains IDE, you can join the waitlist._
What did I just pay for? I want out and feel scammed. I can't cancel Copilot subscription from Github (or that information is very well hidden) and I cannot cancel the payment within Paypal because the payment is probably not in before the first transaction happens. I hate trial offers because you are required to provide payment information up front. Then you forget. I like try first, pay later if you like it. Just disable it for non paying customers.

**Reply by Avec112** [reply]
I actually found a way out from the Trial Subscription. 
https://github.com/settings/billing/summary#usage

**Comment by zozoheir** [comment]
What's taking so long to get this working. There are so many teams out there that build AI apps from scratch, and you can't event add this one feature to the IDE.... I'm switching to VSCode.

**Reply by Avec112** [reply]
I love Intellij to much to switch. I rather use their AI-Assistant but I wanted to tryout Copilot. Another time.

**Comment by SamSpiri** [comment]
JetBrains: Github Copilot plugin is a 3rd party tool developed by Github itself and is not affiliated with JetBrains, and we are not providing waitlists for 3rd party services.

So it's about Microsoft decision. Not stupidity, but intentional and unfair. Don't forget M$ MustDie

**Comment by AndKe** [comment]
@SamSpiri  The "Microsoft decision" works as intended, at least three of my coworkers switched from PyCharm to VSCODE due to the availability of copilot.  
Evil empire.

**Comment by RagingLeviathan** [comment]
I used it for a week or two in PhpStorm, and well... Not really impressed. Feels half-baked, and didn't work half the time. Final straw came when it turns out that using the Copilot chat somehow or another led to my own Windows AD account getting locked out repeatedly? I can't even _pretend_ to know how the hell such a thing happened, I just know it was some combination of PhpStorm & the Copilot Chat.

Absolutely sticking to VSCode for any Copilot stuff! Miserable experience!
